src/Broadcasting/src/AuthorizationStatus.php

```diff
@@ -8,21 +8,12 @@
 
 final class AuthorizationStatus
 {
-    private bool $success;
-    private array $topics;
-    private array $attributes;
-    private ?ResponseInterface $response;
-
     public function __construct(
-        bool $success,
-        array $topics,
-        array $attributes = [],
-        ?ResponseInterface $response = null
+        private readonly bool $success,
+        private readonly array $topics,
+        private readonly array $attributes = [],
+        private readonly ?ResponseInterface $response = null
```

Comment:
These properties can be public.
@butschster what do you think about getters removing?

Reply:
+

Reply:
Done